README.md

```diff
@@ -67,7 +67,11 @@ Datalog 2.0, Philadelphia, PA, June 4-5, 2019.
 
 ## Installing from sources
 
+To install the required dependencies for buidling DDlog run `. ./install-dependencies.sh`
```

Comment:
The comment should mention that manual installation is required if the user has Haskell or Rust or FlatBuffers already installed and want to keep their existing versions. Alternatively we need more complex logic to handle existing compilers.